autoload/lsp/symbol.vim

```diff
@@ -101,7 +101,20 @@ def JumpToWorkspaceSymbol(popupID: number, result: number): void
 	exe $'confirm edit {symTbl[result - 1].file}'
       endif
     else
-      winList[0]->win_gotoid()
+      var wid = fname->bufwinid()
+      if wid != -1
+        # if had one in cur tab
+        # and cur win is same buf
+        if bufwinid(bufnr()) == wid
+          # do nothing
+        else
+          # or pick up one in cur tab
+          wid->win_gotoid()
```

Comment:
Is there a need to check bufwinid(bufnr()) == wid?  If we always do wid->win_gotoid() instead, what will not work?

Reply:
`bufwinid(bufnr()) == wid` is to explain : should not jump if cur win had the same buf then do not jump,
which to be clear just let it be jumped when cur window it is Not same buf.

Q:
> If we always do wid->win_gotoid() instead, what will not work?

A: user perhaps had opened same buf in some windows in cur tab (or not), if just win->win_gotoid() your cursor maybe dumbly fly to another window, that's annoyed and weird, since cur one it is already!